feat: added scroll to top cta

preda-bogdan opened the pull request:
### Summary
<!-- Please describe the changes you made. -->
Added new upsell control for Scroll To Top CTA.


### Will affect the visual aspect of the product
<!-- It includes visual changes? -->
YES/NO

### Screenshots <!-- if applicable -->
<details open>
    <summary>Neve Free Scroll To Top Upsell</summary>

![image](https://github.com/Codeinwp/neve/assets/23024731/eb0f1fdb-12bb-40de-b166-210373cb4d50)
</details>

### Test instructions
<!-- Describe how this pull request can be tested. -->
1. On a fresh instance of Neve w/o. Neve Pro
2. Check that inside the Customizer on **Layout** > **Scroll To Top** the upsell is present, with disabled controls and the upsell link works.

## Check before Pull Request is ready:

* [ ] I have [written a test](CONTRIBUTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](CONTRIBUTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](CONTRIBUTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](CONTRIBUTING.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](CONTRIBUTING.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

<!-- Issues that this pull request closes. -->
Closes Codeinwp/neve-pro-addon#2734.
<!-- Should look like this: `Closes #1, #2, #3.` . -->

irinelenache commented:
@preda-bogdan Tested and everything works fine, except for how the range controls look when Neve Pro is not active
https://vertis.d.pr/i/tcwyPY

preda-bogdan commented:
@irinelenache can you let me know what browser you used? Thank you!

preda-bogdan commented:
@irinelenache Sorry for that, there was a typo in CSS that I introduced when changing the target. It should look correct now.
I also updated the screenshot.

irinelenache commented:
@preda-bogdan Everything's fine now, thank you 🚀
